README.md

```diff
@@ -39,9 +39,9 @@ No extra dependencies needed to use the code!
 
 Add Live Server VSCode extension to deploy a local host server in order to start using it!
 
-<!-- ### Live Demo Server -->
+### Live Demo Server
```

Comment:
- [ ] After cloning what the user will do, please mention it in your README. please write all the steps after cloning the app.

Hint:
```
cd yourrepo
npm install
npm start
```